src/hotspot/share/oops/constMethod.cpp

```diff
@@ -436,6 +435,7 @@ void ConstMethod::print_on(outputStream* st) const {
     m->print_value_on(st);
   }
   st->cr();
+  st->print(" - flags:             "); _flags.print_on(st);               st->cr();
```

Comment:
Method prints its flags as an int and in decoded form, but ConstMethod only prints the decoded form. Any particular reason for this difference?

Reply:
No reason for this difference.  The only reason I added the int form for MethodFlags was because AccessFlags were printed also with the int form.

Reply:
I fixed the constMethod printing with the last commit.